content/en/docs/tasks/network/extend-service-ip-ranges.md

```diff
@@ -0,0 +1,182 @@
+---
+reviewers:
+- thockin
+- dwinship
+min-kubernetes-server-version: v1.29
+title: Extend Service IP Ranges
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state state="alpha" for_k8s_version="v1.29" >}}
+
+This document shares how to extend the existing Service IP range assigned to a cluster.
+
+
+## {{% heading "prerequisites" %}}
+
+{{< include "task-tutorial-prereqs.md" >}}
+
+{{< version-check >}}
+
+<!-- steps -->
+
+## API
+
+Kubernetes clusters that are created with the `MultiCIDRServiceAllocator`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) will create a new ServiceCIDR object with the well-known name `kubernetes` from the kube-apiserver `--service-cluster-ip-range` flag values.
+
+```sh
+kubectl get servicecidr
+```
+```
+NAME         CIDRS          AGE
+kubernetes   10.96.0.0/28   17d
+```
+
+The well-known `kubernetes` Service, that exposes the kube-apiserver endpoint to the Pods, uses as ClusterIP the first IP from the default ServiceCIDR.
+
+```sh
+kubectl get service kubernetes
+```
+```
+NAME         TYPE        CLUSTER-IP   EXTERNAL-IP   PORT(S)   AGE
+kubernetes   ClusterIP   10.96.0.1    <none>        443/TCP   17d
+```
+
+The default Service, in this case, uses the ClusterIP 10.96.0.1, that has the corresponding IPAddress object.
+
+```sh
+kubectl get ipaddress 10.96.0.1
+```
+```
+NAME        PARENTREF
+10.96.0.1   services/default/kubernetes
+```
+
+The ServiceCIDRs are protected with finalizers, to avoid leaving Service ClusterIPs orphans,
+the finalizer is only removed if there is another subnet that contains the existing IPAddresses or
+there are no IPAddresses belonging to the subnet.
+
+## Extend the number of available IPs for Services
+
+There are cases that users will need to increase the number addresses available to Services, previously, increasing the Service range was a disruptive operation that could also cause data loss. With this new feature users only need to add a new ServiceCIDR to increase the number of available addresses.
+
+[![asciicast](https://asciinema.org/a/618594.svg)](https://asciinema.org/a/618594)
```

Comment:
Please avoid referencing an external object here as they are hard to maintain and to localize.

You could use a YouTube video posted to a channel that the Kubernetes project controls, or you could skip it.